extension/scripts/global/globalData.js

```diff
@@ -278,6 +281,16 @@ const DEFAULT_STORAGE = {
 			showAdvanced: new DefaultSetting({ type: "boolean", defaultValue: false }),
 		},
 		containers: new DefaultSetting({ type: "object", defaultValue: {} }),
+		jail: {
+			activity: new DefaultSetting({ type: "array", defaultValue: [] }),
+			faction: new DefaultSetting({ type: "string", defaultValue: "" }),
+			timeStart: new DefaultSetting({ type: "string", defaultValue: "0" }),

```

Comment:
Use the proper type for those, they are `number`s, with `string` they'll get reset to default every time.

Reply:
Changed.